pkg/controller/datavolume/conditions.go

```diff
@@ -177,3 +178,12 @@ func getPVCCondition(anno map[string]string) *cdiv1.DataVolumeCondition {
 	}
 	return nil
 }
+
+func getPrimeMessage(pvc *corev1.PersistentVolumeClaim) string {
+	val, exists := pvc.GetAnnotations()[cc.AnnPVCPrimeName]
+	if exists {
+		pvcPrimeMessage := fmt.Sprintf(" [prime PVC %s]", val)
```

Comment:
nit: why put the space here? This is forcing anyone that is calling this function to take the space into account. Right now not a big deal though. Now of course if you remove the space here, you have to add it to the callers.